.concourse/pipeline.yml

```diff
@@ -1,4 +1,13 @@
 ---
+resource_types:
+- name: pull-request
+  source:
+    repository: teliaoss/github-pr-resource
```

Comment:
As a performance improvement, you can use `tag: latest` to avoid Concourse doing a bunch of semver logic to figure out what the latest image is.